lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc

```diff
@@ -149,23 +149,14 @@ function dosomething_helpers_variable_form($form, &$form_state, $node) {
       '#default_value' => $vars['goal'],
     );
 
-    // @TODO: move this into a function.
-    // If the progress set is higher than the var, save the var as higher.
-    // This allows for campaigns run with external partners to account for
-    // counts submitted directly to the third party, but haven't been
-    // reported back on our site
-    if ($vars['progress'] > $rb_progress) {
-      dosomething_helpers_set_variable('node', $node->nid, 'sum_rb_quanity', $vars['progress']);
-    }
     // Get the progress from the set variable, else do a query for it.
     $rb_progress = dosomething_helpers_get_variable('node', $node->nid, 'sum_rb_quanity') ?: dosomething_reportback_get_reportback_total_by_nid($node->nid);
 
-    $form['goals']['sum_rb_quanity'] = array(
+    $form['goals']['sum_rb_quanity_override'] = array(
```

Comment:
This is picky picky too but the OCD in me cannot not point out this spelling mistake. Not sure how much of a lift it'd be to rename...